src/messaging/ExchangeContext.h

```diff
@@ -90,7 +90,7 @@ class DLL_EXPORT ExchangeContext : public ReferenceCounted<ExchangeContext, Exch
      *
      *  @param[in]    msgType       The message type of the corresponding protocol.
      *
-     *  @param[in]    msgPayload    A pointer to the PacketBuffer object holding the CHIP message.
+     *  @param[in]    msgPayload    A handle to the PacketBuffer object holding the CHIP message.
```

Comment:
The actual function signature was not changed here... Should this change still happen?

Reply:
Undone